### ISSUE TITLE
adding support for specifying cache-control

### ISSUE THREAD

**Issue by jamesdixon** [duizendnegen/ember-cli-deploy-azure-blob#3]
As far as I can tell, Azure doesn't place any sort of `cache-control` header on any blob content. Having `cache-control` headers can be a huge benefit. What are your thoughts on adding an option to apply `cache-control` policies to files that are uploaded?

Here's an example of what the config might look like:

```javascript
env['azure-blob'] = {
  'cache-control': {
    extensions: [
      { extension: 'js', policy: 'max-age=32123311' },
      { extension: 'css', policy: 'max-age=32123311' }
    ],
    files: [
      { file: 'path/to/file', policy: 'max-age=32123311' },
      { file: 'path/to/file', policy: 'private, max-age=23423442' }
    ]
  }
}
```

The idea is to be able to provide default caching policies for certain types of files with the ability to override/set a policy for specific files.

More than happy to work on something, but wanted to get your thoughts first.

**Comment by duizendnegen** [comment]
I am :+1: on this.
(Naming should be cacheControl instead of cache-control as per the config camel casing, but that's details)

**Comment by jamesdixon** [comment]
Awesome. Let me know if I can be of help.

**Comment by tomasbasham** [comment]
Has any progress been made on this?

**Comment by duizendnegen** [comment]
None by me - I'm welcoming a PR

**Comment by tomasbasham** [comment]
Ok. I can implement this. Hopefully will have something next week.

**Comment by duizendnegen** [comment]
Closed through #6, will release new tag later today.

**Comment by tomasbasham** [comment]
Fantastic!
